plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoProjectionPushdownPlans.java

```diff
@@ -0,0 +1,295 @@
+/*
```

Comment:
Can we have the plan assertion as a part of `ConnectorTest/ConnectorSmokeTest`

Reply:
This is similar to what `TestDeltaLakeProjectionPushdownPlans` and other connectors' Projection Pushdown Plan tests.